client/styles/components/_loader.scss

```diff
@@ -0,0 +1,46 @@
+$color_1: #000000;
```

Comment:
you should load the color from the appropriate theme, see `client/styles/abstracts/_variables.scss`, and use the SASS function `getThemifyVariable`